### ISSUE TITLE
Can't connect Facebook if using New App

### ISSUE THREAD

**Issue by UVLabs** [Codeinwp/tweet-old-post#154]
@selul The plugin doesn't connect to new facebook apps and shows this error:

https://pasteboard.co/HiraBSv.png
![image](https://user-images.githubusercontent.com/10324144/39325587-9cdb957c-4960-11e8-9da7-de6d0f3e4874.png)

It doesn't make sense because this is how my app domains looks with all possible domains:

![image](https://user-images.githubusercontent.com/10324144/39325561-8c24f3ae-4960-11e8-8e59-029f5ecaaf48.png)

I also have the URLs entered in the Valid Oauth login area:

![image](https://user-images.githubusercontent.com/10324144/39325637-b62734be-4960-11e8-9d26-fc5d26772e5b.png)

The redirect validator says things are fine:

![image](https://user-images.githubusercontent.com/10324144/39325694-d77576e4-4960-11e8-9b5a-5cf294fe8a81.png)

There might be an issue in the api code maybe, in older fb apps running an older API example v2.8 this does not happen.

Sidenote: On older apps when I connected my facebook it still did not let me choose a page, but rather just defaulted to my Facebook account, I mentioned this in my tests

**Comment by selul** [comment]
@UVLabs yep, it seems we had a nasty issue with this, but i've fixed it.
